pkg/controller/core/workload_controller.go

```diff
@@ -607,9 +608,16 @@ func (r *WorkloadReconciler) Create(e event.CreateEvent) bool {
 	workload.AdjustResources(ctx, r.client, wlCopy)
 
 	if !workload.HasQuotaReservation(wl) {
-		err := r.queues.AddOrUpdateWorkload(wlCopy)
-		if err != nil {
-			log.V(2).Info(fmt.Sprintf("%s; ignored for now", err))
+		if err := r.queues.AddOrUpdateWorkload(wlCopy); err != nil {
+			var msg string
+			baseMsg := "for workload didn't exist or not active; ignored for now"
+			switch {
+			case errors.Is(err, queue.ErrClusterQueueDoesNotExist):
+				msg = fmt.Sprintf("ClusterQueue %s", baseMsg)
+			case errors.Is(err, queue.ErrQueueDoesNotExist):
+				msg = fmt.Sprintf("LocalQueue %s", baseMsg)
+			}
+			log.V(2).Info(msg)
```

Comment:
Hm `ErrClusterQueueDoesNotExist` already contains the information this is about "clusterQueue":  https://github.com/kubernetes-sigs/kueue/blob/cc65f7e7701df31e77959ef664384616abd54f9e/pkg/queue/manager.go#L42-L43. So why to duplicate the information?

Also, this is a bulk of duplicated code, please commonize.

Reply:
That makes sense. Here, key problem is dropped the "inactive" reason from logging message and mention "queue" instead of localQueue.

https://github.com/kubernetes-sigs/kueue/pull/3605/files#diff-60dd240c20adbd6a189d018d1c216c2d296730f446c341d8bf449fa6657964ffL733

So, I will remove these error-typed checkings and then add an "inactive" reason.

Reply:
Done.
Please let me know if you have more recommendations.
Thanks.

Reply:
I see, but do you have a code reference (could be a unit test) indicating that the error is returned for inactive queues (this is something I'm not sure about).

Reply:
We add LocalQueue to the queue manager only if it is active.

https://github.com/kubernetes-sigs/kueue/blob/5ad7e23fdeed50d90d90629c28623f592063ac13/pkg/controller/core/localqueue_controller.go#L141-L146

Reply:
I see, so the message "does not exist or inactive" is accurate.  Do we have any unit tests for this scenario?

How about ClusterQueue? In case it is inactive we still expect it in cache here: https://github.com/kubernetes-sigs/kueue/blob/5ad7e23fdeed50d90d90629c28623f592063ac13/pkg/cache/cache.go#L333-L340

Reply:
Agree. For CQ we need to revert.

Reply:
As I discussed with @mimowo, for LocalQueue, the "inactive" reason is valid. But for ClusterQueue, we always insert that to queueManager, and then we identify the active CQs by `Active` function: https://github.com/kubernetes-sigs/kueue/blob/5ad7e23fdeed50d90d90629c28623f592063ac13/pkg/queue/cluster_queue.go#L58

So, we decided to remove the "inactive" reason only from ClusterQueue.

Reply:
Done